app/helpers/search_helper.rb

```diff
@@ -0,0 +1,230 @@
+# frozen_string_literal: true
+
+# rubocop:disable  Rails/HelperInstanceVariable - extensive use of @query in these helpers
```

Comment:
@joshmcarthur while it's fine... I don't love the use of @query here. Not sure if this is existing code or new code, but would be good if there is a way to make this more boring, especially given how long the file is

Reply:
@mischa-s have a read of the description in the PR - refactoring existing behaviour in this app is out of scope. I don't love it either, but it's what is there, and it's not that trivial of a refactor to change all the call sites for these methods to pass in `@query`.

Reply:
And to answer your question - yes, it's existing code.